server/src/internalClusterTest/java/org/opensearch/search/nested/SimpleNestedIT.java

```diff
@@ -487,7 +487,8 @@ public void testExplain() throws Exception {
             )
             .setRefreshPolicy(IMMEDIATE)
             .get();
-        indexRandomForConcurrentSearch("test");
+
+        indexRandomForMultipleSlices("test");
```

Comment:
adding this function as the bogus docs will be added in both the cases concurrent/non-concurrent case and the range of the doc ids will be different with each run in both the cases.